### PR TITLE
Add integration tests with selenium

### PR DESCRIPTION
**Pull request by cskaandorp**
Using selenium to conduct integration test

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -167,3 +167,4 @@ sketch
 notes.md
 
 flask_config.json
+flask_config.toml
```
Comment (J535D165) [inline .gitignore:170]:
```suggestion
flask_config.*
```

File: .github/workflows/ci-frontend.yml [hunk 2]
```diff
@@ -28,6 +28,6 @@ jobs:
           pip install pytest
           pip install pytest-random-order
           pip install --no-cache-dir .
-      - name: Test flask app
+      - name: Test flask web app
         run: |
-          pytest --random-order asreview/webapp/tests
+          pytest --random-order --ignore=asreview/webapp/tests/integration_tests asreview/webapp/tests
```
Comment (J535D165) [inline .github/workflows/ci-frontend.yml:33]: What about configuring this in `pytest.ini`? It might be annoying for users to run this by default.

Comment (cskaandorp) [reply .github/workflows/ci-frontend.yml:33]: It didn't work! But maybe I wasn't sharp.

Comment (J535D165) [inline .github/workflows/ci-frontend.yml:33]:
```suggestion
          pytest --random-order asreview/webapp/tests
```

This should work now with the pytest.ini file.

**Review by J535D165 — COMMENTED** [review]
It looks good to me!

**Comment by J535D165** [comment]
Thanks!! Very nice!

**Pull request merged**